srtcore/epoll.cpp

```diff
@@ -243,6 +299,23 @@ int CEPoll::update_usock(const int eid, const SRTSOCKET& u, const int* events)
     evts &= ~SRT_EPOLL_ET;
     if (evts)
     {
+        if (!edgeTriggered)
```

Comment:
This implements the "edge-triggered only" events. This is the case for the newly introduced event type, `SRT_EPOLL_UPDATE`. Events that are "edge only" are collected in `SRT_EPOLL_ETONLY` constant.